sql-gremlin/README.asciidoc

```diff
@@ -258,7 +258,7 @@ Currently, `COUNT( [DISTINCT] numeric)`, `COUNT( [DISTINCT] value)`, and `SUM( [
 
 Additional Limitations
 ----------------------
-
+* Currently JDBC driver supports [Tableau Data Extracts (TDE)](https://www.tableau.com/about/blog/2014/7/understanding-tableau-data-extracts-part1) and has limitations which may prevent or significantly limit functionality when using Live Connection in Tableau.
```

Comment:
asciidoc link should be in this format: 
`https://www.tableau.com/about/blog/2014/7/understanding-tableau-data-extracts-part1[Tableau Data Extracts (TDE)]`
The link in front without round brackets.

Reply:
Fixed